web-ui/src/main/resources/catalog/locales/en-search.json

```diff
@@ -419,5 +419,52 @@
   "mdActions": "Available actions",
   "mdActions-view": "Viewable",
   "mdActions-download": "Downloadable",
-  "mdActions-process": "Processable"
+  "mdActions-process": "Processable",
+  "GUFrate": "Rating",
+  "GUFnoRatings": "No ratings <span class=\"fa fa-fw fa-star\">&nbsp;</span>",
+  "GUFfromRatingCountRatings": "from {{ratingCount}} ratings",
+  "GUFwaitingForApproval": "Waiting for approval",
+  "GUFseeAllFeedbacks": "See all feedbacks",
+  "GUFcommentsLastCommentOn": "{{userfeedbackCount}} comments, last comment on {{date}}.",
+  "GUFcountReviews": "{{userfeedbackCount}} Reviews",
+  "GUFdeleteUserFeedback": "Delete",
+  "GUFpublishUserFeedback": "Publish",
+  "GUFclose": "Close",
+  "GUFsave": "Save",
+  "GUFfeedbackOn": "Feedback on:",
+  "GUFnoComments": "No comments",
+  "GUFaddRating": "Review this dataset",
+  "GUFplusAddRating": "<span class=\"fa fa-plus\"></span> Review this dataset",
```

Comment:
We usually put the HTML in templates not in translation to keep translation as easy as possible